### ISSUE TITLE
xgb predict class

### ISSUE THREAD

**Issue by daxiongshu** [dmlc/xgboost#477]
Hi there, is there a function in xgb to predict class instead of probability? Like the predict() of sklearn's RandomForestClassifier.

Thank you very much!

**Comment by tqchen** [comment]
Originally, the softmax(instead of softprob) loss function do predicts the class. I think it is also supported in the XGBClassifier sklearn interface. In either way, you can do np.argmax to get the class index
